lib/PuppeteerSharp/EmulationManager.cs

```diff
@@ -1,21 +1,29 @@
-﻿using System;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
 using PuppeteerSharp.Media;
 using PuppeteerSharp.Messaging;
 
 namespace PuppeteerSharp
 {
     internal class EmulationManager
     {
+        #region Members
         private readonly CDPSession _client;
         private bool _hasTouch;
         private bool _emulatingMobile;
+        #endregion Members
```

Comment:
I'm trying to get out of the `#region` blocks. The community is not using them anymore.

Reply:
Wasn't aware of this. Thanks for letting know :)